README.md

```diff
@@ -1,129 +1,40 @@
 | image | The docker image to use | `string` | `"bitnami/kafka:latest"` | no |
 | instance\_type | The instance type to use | `string` | `"t3.large"` | no |
 | iops | IOPS to provision for EBS storage | `number` | `500` | no |
+| key\_store | Akey store for SSL | <pre>object(<br>    { keystore = string,<br>    password = string }<br>  )</pre> | n/a | yes |
+| mm2\_properties | mm2 properties file path | `string` | n/a | yes |
 | nodes | Number of nodes | `number` | `1` | no |
 | private\_key | Private key for SSH access | `string` | n/a | yes |
+| tld | The tld for your host default is a dev | `string` | `"dev"` | no |
+| trust\_store | Akey store for SSL | <pre>object(<br>    { truststore = string,<br>    password = string }<br>  )</pre> | n/a | yes |
 | user | LDAP user to use for connections | `string` | n/a | yes |
 | user\_groups | User groups to assign to cluster | `list(string)` | `[]` | no |
 | volume\_size | The volume size to use in GB | `number` | `50` | no |
-| trust\_store| The trust store object for kafka (see below for more details) | `object` | none | yes |
-| key\_store | The key store object for kafka(see below for more details) | `object` | none | yes |
-| mm2\_properties| location of the mm2.properties file  | `string` | none | yes |
-
-
-
-## Key Store object
-This object has two properties that needs to be filled
-| Name | Description | Type | Default | Required |
-|------|-------------|------|---------|:--------:|
-| keystore | The path of the keystore file in JKS format| `string` | none | yes |
-| password | The password to be used for the key store | `string` | none | yes |
-
-## trust Store object
-This object has two properties that needs to be filled
-| Name | Description | Type | Default | Required |
-|------|-------------|------|---------|:--------:|
-| truststore | The path of the truststore file in JKS format| `string` | none | yes |
-| password | The password to be used for the trust store | `string` | none | yes |
-
-## mm2.properties
-Here is an example mm2.properties file
-```
-clusters = hsdp, openline
-hsdp.bootstrap.servers=172.30.16.195:8282
-openline.bootstrap.servers=ec2-3-237-7-189.compute-1.amazonaws.com:80
-hsdp->openline.emit.heartbeats = false
-hsdp->openline.enabled=true
-openline->hsdp.enabled=false
-config.storage.replication.factor = 1
-offset.storage.replication.factor = 1
-status.storage.replication.factor = 1
-replication.factor = 1
-
-```
-
-Do not add the below keys to your mm2.properties file, these are auto added. 
-```
-security.protocol = 
-ssl.endpoint.identification.algorithm =
-ssl.truststore.location = /bitnami/kafka/config/certs/truststore.jks
-ssl.truststore.password = ******
-ssl.truststore.type = JKS
-ssl.keystore.location = /bitnami/kafka/config/certs/mm.keystore.jks
-ssl.keystore.password = ****
-```
-
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
-| kafka\_nodes | Container Host IP addresses of Kafka Mirror maker instances |
```

Comment:
can you kindly merge the Readme rather than replace

Reply:
oops, I overwrote because the file was empty. Reverted